app/src/main/java/com/example/dashcarr/presentation/tabs/settings/maps_settings/adapter/PointOfInterestRecyclerAdapter.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.dashcarr.presentation.tabs.settings.maps_settings.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.dashcarr.R
+import com.example.dashcarr.domain.entity.PointOfInterestEntity
+
+class PointOfInterestRecyclerAdapter (private val onPointListener: OnPointClickListener)
```

Comment:
That seems like a really important concept for working with the database. I don't know it and I assume the others does't now it as well. Can you give us tomorrow a short introduction for the best practices if you want to read/write to the database?